### DISCUSSION TITLE
New contributor looking to help

### DISCUSSION THREAD

**Discussion by TeoSkondras** [MovingBlocks/Terasology#5100] [Q&A] [answered]
Hello! I would love to contribute to your project as a Minecraft fan. Is there anything that my help can be useful?

**Comment by jdrueckert** [comment, marked as answer]
Hi @TeoSkondras,
have a look at our website on how to get started with contributing to Terasology: https://terasology.org/contribute/
You'll also find links to good first issues and ongoing efforts you could consider helping with.
Other than that, you can also join us on our [Discord](https://discordapp.com/invite/Terasology) where we can talk in more detail about your skillset, coding experience, and contribution area of interest so that we can find a suitable spot for you to help :slightly_smiling_face:
